### DISCUSSION TITLE
docJar fails to generate links to standard library

### DISCUSSION THREAD

**Discussion by TheElectronWill** [com-lihaoyi/mill#1479] [Q&A]
## The problem

Links to standard Java and Scala classes aren't generated properly by `docJar`, resulting in a ScalaDoc that contains plain text instead of a proper link. Linking to a class in the project does work.

Should I add some docJar options to fix this? I feel like this should work out-of-the box, if possible.

## Reproducible example
build.sc
```scala
import mill._, scalalib._

object foo extends ScalaModule {
  def scalaVersion = "2.12.6"
}
```
foo/src/foo/bar/Test.scala
```scala
package foo.bar

import java.nio.ByteBuffer

/** ==ScalaDoc test==
  *  - Link to the standard Scala library: [[scala.collection.Seq]]
  *  - Link to the standard Java library: [[java.util.List]]
  *  - Link to an imported class: [[ByteBuffer]]
  */
class Test {}
```

Output:
```
model contains 4 documentable templates
foo/src/foo/bar/Test.scala:5: warning: Could not find any member to link for "scala.collection.Seq".
/** ==ScalaDoc test==
^
foo/src/foo/bar/Test.scala:5: warning: Could not find any member to link for "java.util.List".
/** ==ScalaDoc test==
^
foo/src/foo/bar/Test.scala:5: warning: Could not find any member to link for "ByteBuffer".
/** ==ScalaDoc test==
^
```

**Comment by Sailsman63** [comment]
I know this is an old question. If it's still of interest, note that vanilla command-line javadoc doesn't do this either unless you explicitly `-link` it.
